website/src/content/api-reference/commercetools-frontend-application-shell.mdx

```diff
@@ -113,17 +113,33 @@ const EntryPoint = () => {
 
 ## createApolloClient
 
-Creates a new instance of the Apollo Client. Use this if you need to [configure the Apollo cache](https://www.apollographql.com/docs/react/caching/cache-configuration).
+Creates a new instance of the Apollo Client. Use this to extend certain functionalities of the preconfigured Apollo Client.
 
 ```js
+import { createApolloClient } from '@commercetools-frontend/application-shell';
+
 createApolloClient({
-  cache: {
-    // ...
-  },
+  // ...
 })
 ```
 
-We recommend creating a separate file, for example `src/apollo-client.js`, and define the configuration there.
+Available options are:
+
+* `cache` (optional): Configuration of the [Apollo cache](https://www.apollographql.com/docs/react/caching/cache-configuration) in relation to the data requirements of your Custom Application.
+
+* `restLink` (optional): Instance of the [Apollo REST link](https://www.apollographql.com/docs/react/api/link/apollo-link-rest/).
+
+  <Info>
+
+  This feature is available from version `21.10.0` onwards.
+
+  The `apollo-link-rest` and its related dependencies are not included in the `@commercetools-frontend/application-shell` package and must be installed separately.
+
+  When configuring the REST link, we recommend setting the `uri` using the `getMcApiUrl()` utility function.
```

Comment:
nit: perhaps we could explain this utility function is also exported from this package.